gateway/fedimint-gateway-server/src/client.rs

```diff
@@ -140,19 +141,21 @@ impl GatewayClientBuilder {
                 .await
                 .map_err(AdminGatewayError::ClientCreationError)?;
             let db = Database::new(rocksdb, ModuleDecoderRegistry::default());
-            let root_secret = self.client_plainrootsecret(&db).await?;
+            let root_secret = RootSecret::Custom(self.client_plainrootsecret(&db).await?);
```

Comment:
@m1sterc001guy Can you verify if this looks good? The old version was not doing any extra derivation, while the new one is doing the same standard one that all apps are using, right?

Reply:
Looks equivalent to me, but can you please confirm @m1sterc001guy?

Reply:
Sorry for the delay. Yes this is correct. Only affects old gateways. Looks good to me